solutions/provisioning_azure_azuredevops/index.asciidoc

```diff
@@ -159,75 +154,10 @@ resources:
 ```
 
-Implicit Chaining for *orchestration* is possible by using trigger condition. Calling pipelines explicitly is so far only possible with scripting. The code snippet below shows an example:
-```Powershell
-#
-# Make call to schedule pipeline run
-#
-
-# Body
-$body = @{
-    stagesToSkip = @()
-    resources = @{
-        self = @{
-            refName = $branch_name
-        }
-    }
-    templateParameters = $params
-    variables = @{}
-}
-$bodyJson = $body | ConvertTo-Json
-# Uri extracted from the Azure DevOps UI
-# $org_uri and $prj_id contain names of current organization/ project
-# $pl_id denotes the internal pipeline id to be started
-$uri = "${org_uri}${prj_id}/_apis/pipelines/${pl_id}/runs?api-version=5.1-preview.1"
-
-# Output paramters
-Write-Host("--------  Call ${pl_name} --------")    
-Write-Host("Headers: ${headersJson}")
-Write-Host("Json body: ${bodyJson}")    
-Write-Host("Uri: ${uri}")    
-
-try 
-{
-    # Trigger pipeline
-    $result = Invoke-RestMethod -Method POST -Headers $headers -Uri $uri -Body $bodyJson
-    Write-Host("Result: ${result}")        
-
-    # Wait until run completed
-    $buildid = $result.id
-    $start_time = (get-date).ToString('T')
-    Write-Host("------------ Loop until ${pl_name} completed --------")
-    Write-Host("started runbuild ${buildid} at ${start_time}")   
-    
-    # Uri for checking state
-    $uri = "${org_uri}${prj_id}/_apis/pipelines/${pl_id}/runs/${buildid}?api-version=5.1-preview.1"
-
-    Do {
-        Start-Sleep -Seconds 60
-        $current_time = (get-date).ToString('T')
-
-        # Retrieve current state
-        $result = Invoke-RestMethod -Method GET -Headers $headers -Uri $uri
-        $status = $result.state
-        Write-Host("Received state ${status} at ${current_time}...")
-    } until ($status -eq "completed")
-
-    # return result
-    $pl_run_result = $result.result
-    Write-Host("Result: ${pl_run_result}")   
-    return $pl_run_result
-}
-catch { 
-    $excMsg = $_.Exception.Message
-    Write-Host("Exception text: ${excMsg}")
-    return "Failed"
-}
-```
-Orchestration must take dependencies into account. They might result from the deployed code or the scope of the pipeline (scope is e.g. a single microservice and code includes the libraries needed).
-Orchestrated pipelines must pass data between them. The recommended method is to use key vault. 
+Implicit Chaining for *orchestration* is possible by using trigger condition. Calling pipelines explicitly is so far only possible with scripting. 
 
-*Recreation of resources in short intervals* might cause pipelines to fail. Even if resources are deleted they might still exist in the background (even although soft delete is not applicable). Programming languages can therefore get confused if pipelines recreate things in short intervals. Creating a new resource group can solve the problem since they are part of the tecnical resource id.
+[.internal]
+provisioning_azure_devops_orch
```

Comment:
Internal snippets that are used in solutions currently must be prefixed with `solution_`, because of technical reasons in a synchronization action.